query/build.gradle

```diff
@@ -0,0 +1,40 @@
+// Build instructions for standalone Java library.
+
+apply plugin: 'java'
+
+// Set the source and target to Java 6.
+tasks.withType(JavaCompile) {
+    options.compilerArgs << "-source" << "1.6" << "-target" << "1.6"
+}
+
+dependencies {
+    compile project(':core')
+    compile project(':java')
+//    testCompile project(':core')
```

Comment:
Grade question - the testCompile project(':core').sourceSets.test.output line seems to give me the following error sometimes when I run tests with JUnit from inside Android Studio, since the Core sourceSets do not exist yet sometimes. This doesn't happen when I run with Gradle (even within Android Studio). Not sure if this is a problem.

Error:(18) A problem occurred evaluating project ':query'.
> Could not find property 'sourceSets' on project ':core'.